src/Ionide.ProjInfo/Utils.fs

```diff
@@ -1,19 +1,32 @@
 namespace Ionide.ProjInfo
 
+module internal Paths =
+    /// provides the path to the `dotnet` binary running this library, duplicated from
+    /// https://github.com/dotnet/sdk/blob/b91b88aec2684e3d2988df8d838d3aa3c6240a35/src/Cli/Microsoft.DotNet.Cli.Utils/Muxer.cs#L39
+    let dotnetRoot =
+        System
+            .Diagnostics
+            .Process
+            .GetCurrentProcess()
+            .MainModule
+            .FileName
```

Comment:
Why not just one line?

Reply:
fantomas formatting, just letting it do its thing until we settle on specific config settings.